### ISSUE TITLE
@charset support

### ISSUE THREAD

**Issue by nerkles** [less/less.ruby#66]
use of @charset rule causes syntax error.

Rule is documented here: 
http://www.w3.org/International/questions/qa-css-charset

**Comment by samalone** [comment]
I also ran into this. Dreamweaver CS4 adds @charset "UTF-8"; to the top of new CSS files by default, so this causes problems when converting Dreamweaver CSS files to less.

**Comment by gphat** [comment]
Also ran into this.  I even tried to take a crack at it but I'm a Perl guy, not Ruby and couldn't figure it out quickly.  Upvoting! :)

**Comment by carnesmedia** [comment]
+1

**Comment by bitsandbooks** [comment]
Yeah, I'm putting in another vote for **@charset** support. I get an error when I compile my stylesheet with LESS that reads:

> ! Syntax Error: on line 1: expected: got " after:
> @charset

Maybe **@charset** can be made into a reserved (that is to say, _ignored_) LESS rule?

**Comment by cloudhead** [comment]
fixed in less.js, which will make its way to the main release soon.

http://github.com/cloudhead/less.js
